T Sharp.t#

```diff
@@ -0,0 +1 @@
+push "Hello World" print
```

Comment:
@MrBrain295 Next time, check for files not in the right place, like this one.

Reply:
oh, sorry